src/index.html

```diff
@@ -1,63 +1,85 @@
 
     <link rel="shortcut icon" href="./assets/favicon.ico" />
-</head>
-<body>
+  </head>
+  <body>
     <header>
-        <div class="header-container">
-          <div class="left-header">
-            <img
-              src="./assets/daily.svg"
-              class="daily-logo"
-              alt="Daily logo reads Daily"
-            />
-            <strong>
-              <p class="header-title">Daily demo</p>
-            </strong>
-          </div>
-          <div class="right-header">
-            <a
-              href="https://docs.daily.co/reference"
-              target="_blank"
-              class="link"
-            >
-              <button class="button docs-button white-button">
-                API docs
-                <img
-                  class="arrow-icon"
-                  src="./assets/new-tab-icon.png"
-                  alt="Arrow icon points up and to the right"
-                  aria-hidden="true"
-                />
-              </button>
-            </a>
-            <span class="divider"></span>
-            <a href="https://github.com/daily-demos/boilerplate-vanilla-js/" target="_blank">
+      <div class="header-container">
+        <div class="left-header">
+          <img
+            src="./assets/daily.svg"
+            class="daily-logo"
+            alt="Daily logo reads Daily"
+          />
+          <strong>
+            <p class="header-title">Daily Prejoin Presence Demo</p>
+          </strong>
+        </div>
+        <div class="right-header">
+          <a
+            href="https://docs.daily.co/reference"
+            target="_blank"
+            class="link"
+          >
+            <button class="button docs-button white-button">
+              API docs
               <img
-                class="octocat"
-                src="./assets/github.png"
-                alt="GitHub logo Octocat"
+                class="arrow-icon"
+                src="./assets/new-tab-icon.png"
+                alt="Arrow icon points up and to the right"
+                aria-hidden="true"
               />
-            </a>
+            </button>
+          </a>
+          <span class="divider"></span>
+          <a
+            href="https://github.com/daily-demos/boilerplate-vanilla-js/"
+            target="_blank"
+          >
+            <img
+              class="octocat"
+              src="./assets/github.png"
+              alt="GitHub logo Octocat"
+            />
+          </a>
+        </div>
+      </div>
+    </header>
+    <noscript>
+      This application requires JavaScript to run. Please enable JavaScript in
+      your browser.
+    </noscript>
+    <div id="container">
+      <div id="entry">
+        <button id="createRoom">Create Room</button>
+      </div>
+      <div id="call" class="hidden">
+        <div id="callFrame"></div>
+        <div id="sidebar">
+          <div id="invite">Invite URL:
+            <br /><a id="inviteURL" href=""></a></div>

```

Comment:
Nit: It's not super common to use `br` and `hr` anymore (please don't immediately link to where I have recently and forgot lol!) More commonly, you'd use CSS to get the same effect (e.g. if you put `display:block;` on the `<a>` tag, it'll go on a new line, or if you put the "invite url:" text in a `<p>` tag, it will push the link to a new line. (I mentioned this elsewhere, but I'd suggest putting the text in a text tag like a `<p>` tag to make it more semantic.

Reply:
I meant to also include: instead of an hr, you can use a border on the container.
Not a big deal, though.